### ISSUE TITLE
Adding custom aggregators

### ISSUE THREAD

**Issue by 1f604** [silk-framework/silk#70]
Is it possible to add custom aggregators, such as Fellegi-Sunter, naive Bayes, R-Swoosh or SVM?

**Comment by robertisele** [comment]
Yes, it is possible to add new aggregators by implementing the org.silkframework.rule.similarity.Aggregator interface and registering it as a plugin. 
However, the parameters for the aggregators you propose are usually learned, for instance, using a supervised learning algorithm. Integrating a learning algorithm for this would also be possible bu more work.
